src/services/LoggingService.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Logs info and errors to NewRelic and console.
+ *
+ * Requires the NewRelic Browser JavaScript snippet.
+ */
+class LoggingService {
+  static logInfo(message) {
+    if (typeof newrelic !== 'undefined') {
+      newrelic.addPageAction('INFO', { message });
+    }
+  }
+
+  static logError(error) {
+    if (typeof newrelic !== 'undefined') {
+      newrelic.noticeError(error);
+    }
+  }
+
+  static logAPIErrorResponse(error) {
+    let { message } = error;
```

Comment:
Can also do ```const {message, response, request} = error;``` so you don't have to repeat so much.

Reply:
Thanks.  Code copied from edx-portal.  Going to merge to unblock prod.  I like the idea though.